catroid/src/main/java/org/catrobat/catroid/formulaeditor/InternFormulaKeyboardAdapter.java

```diff
@@ -167,6 +167,9 @@
 			case R.string.formula_editor_function_join:
 				return buildDoubleParameterFunction(Functions.JOIN, STRING, "hello",
 						STRING, " world");
+			case R.string.formula_editor_function_join2:
+				return buildTripleParameterFunction(Functions.JOIN2, STRING, "hello",
+						STRING, " world", STRING, "!");
```

Comment:
I just found this code part. Actually, all the textual strings in https://github.com/Catrobat/Catroid/blob/b71d974b33fe16a7688871a6d2a105bd0a872923/catroid/src/main/java/org/catrobat/catroid/formulaeditor/InternFormulaKeyboardAdapter.java#L161-L186 such as here "hello" and "world" should in fact instead be resources in strings.xml that can be translated by our translators. Not sure if you want to do it in this ticket, or whether we make a new one. LMK.

Reply:
Unfortunately I have no time left. Please create a separate ticket.